segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperServiceRunner.java

```diff
@@ -169,4 +190,32 @@ private void cleanupDirectories() throws IOException {
     }
 
     //endregion
+
+    public static void main(String[] args) throws Exception {
+        val b = BookKeeperServiceRunner.builder();
+        b.startZk(false);
+        try {
+            int bkBasePort = Integer.parseInt(System.getProperty(PROPERTY_BASE_PORT));
+            int bkCount = Integer.parseInt(System.getProperty(PROPERTY_BOOKIE_COUNT));
+            val bkPorts = new ArrayList<Integer>();
+            for (int i = 0; i < bkCount; i++) {
+                bkPorts.add(bkBasePort + i);
+            }
+
+            b.bookiePorts(bkPorts);
+            b.zkPort(Integer.parseInt(System.getProperty(PROPERTY_ZK_PORT)));
+            b.ledgersPath(System.getProperty(PROPERTY_LEDGERS_PATH));
+            b.startZk(Boolean.parseBoolean(System.getProperty(PROPERTY_START_ZK, "false")));
+        } catch (Exception ex) {
+            System.out.println(String.format("Invalid or missing arguments (via system properties). Expected: %s(int), " +
+                            "%s(int), %s(int), %s(String). (%s).", PROPERTY_BASE_PORT, PROPERTY_BOOKIE_COUNT, PROPERTY_ZK_PORT,
+                    PROPERTY_LEDGERS_PATH, ex.getMessage()));
+            System.exit(-1);
+            return;
+        }
+
+        BookKeeperServiceRunner runner = b.build();
+        runner.startAll();
+        Thread.sleep(Long.MAX_VALUE);
```

Comment:
There is no future to wait on here. This starts BK in the current process and exits when the process is killed. This is similar to how we start the Controller and SegmentStore services too.